modules/cgrindel_bazel_starlib/0.16.1/MODULE.bazel

```diff
@@ -0,0 +1,137 @@
+module(
+    name = "cgrindel_bazel_starlib",
+    version = "0.16.1",
+    bazel_compatibility = [">=6.1.0"],
+)
+
+# MARK: - Runtime Dependencies
+
+bazel_dep(
+    name = "rules_go",
+    version = "0.40.1",
+    repo_name = "io_bazel_rules_go",
+)
+bazel_dep(
+    name = "gazelle",
+    version = "0.31.1",
+    repo_name = "bazel_gazelle",
+)
+bazel_dep(name = "bazel_skylib", version = "1.4.2")
+bazel_dep(
+    name = "stardoc",
+    version = "0.5.6",
+    repo_name = "io_bazel_stardoc",
+)
+bazel_dep(
+    name = "buildifier_prebuilt",
+    version = "6.0.0.1",
+)
+bazel_dep(name = "platforms", version = "0.0.6")
+
+go_sdk = use_extension("@io_bazel_rules_go//go:extensions.bzl", "go_sdk")
+use_repo(
+    go_sdk,
+    go_sdk = "go_default_sdk",
```

Comment:
No blocker, just curious: Is this still needed and if so, for what?

Reply:
Not sure. I will check.